1-estrategia/README.md

```diff
@@ -1,23 +1,237 @@
+	La respuesta que más puntuación ha obtenido han sido las historias de casos reales, le siguen los consejos y los reportajes. 
+
+-	**¿En qué momento del día accederías a una página web informativa sobre la igualdad de género?**
+	Un 62,5% consumiría el contenido a la noche en la franja horaria entre las 22:00 y las 00:00. 
+
+-	**Quieres contactar con la página web, ¿cómo preferirías ponerte en contacto?**
+	El 62,5% de los encuestados contactarían mediante e-mail.
+
+-	**¿Crees que una página web sobre la igualdad de género es útil? ¿Tiene interés?**
+	Un 50% dice que si es útil y el otro 50% dice que tal vez. 
+
+#### Conclusiones generales:
+- Los **contenidos audiovisuales** son los que tienen más éxito y los contenidos en audio ninguno. 
+
+- Un **buscador avanzado** es imprescindible en una página web y también el contenido que hay en ella.
+
+- Las **historias de casos reales** son el contenido que más interesa en relación a la igualdad de género.
+
+- En el apartado de “atención al cliente”, es imprescindible añadir un **e-mail** para que los clientes se puedan poner en **contacto**, es la forma más demandada. 
+
+
+## Personas
+
+Mediante el uso de personas vamos a representar a los posibles personajes que van a usar el sitio web que vamos a diseñar. De esta forma no solo representamos a una persona sino al conjunto de las mismas, recogemos sus conductas, habilidades, actitudes de nuestros posibles usuarios potenciales. Este proceso nos va a ayudar mucho a la hora de elaborar nuestra interfaz ya que diseñaremos la misma en función de las necesidades de nuestros usuarios. 
+Para ello hemos elaborado tres plantillas y por ende tres personas que representan al conjunto de nuestro posible público objetivo. Ellos son: Juan, Paula y Dolores. 
+
+* **Juan** es estudiante de criminología y este año termina la carrera. Dice no tener problemas a la hora de manejar diferentes sitios webs pero tampoco de considera un hacha.
+* **Paula** es directora informática de BOSCH Bilbao. Le apasiona estar en todo momento informada y le encanta leer sobre temas de desigualdad de género. No tiene ningún tipo de problema a la hora de manejar sitios web, ya que se dedica a ello. 
+* **Dolores** es ama de casa y no controla mucho el mundo web. Se siente segura utilizando la página web de las recetas de MJ ya que le resulta familiar y sencilla. Por lo demás prefiere no tocar mucho no vaya a des configurar el ordenador que tiene en casa. Para estar informada compra todos los días prensa escrita pero no le importaría pasarse plataformas web siempre y cuando sean claras y sencillas.
+
+* <img src="img/JUAN.jpg">
+* <img src="img/PAULA.jpg">
+* <img src="img/DOLORES.jpg">
+
+Hemos decidido que vamos a diseñar nuestra interfaz para Dolores ya que es quién mas problemas va a tener a la hora de navegar en nuestro sitio web. Paula y Juan están muy familiarizados con este entorno y creemos que no tendrán ningún tipo de problema a la hora de navegar en nuestra web. 
```

Comment:
```suggestion
Hemos decidido que vamos a diseñar nuestra interfaz para Dolores ya que es quien más problemas va a tener a la hora de navegar en nuestro sitio web. Paula y Juan están muy familiarizados con este entorno y creemos que no tendrán ningún tipo de problema a la hora de navegar en nuestra web. 
```

Definís a vuestras personas exclusivamente en función de su hábito de consumo de internet; no mencionáis nada acerca de su interés y motivación en relación a vuestro tema, ni sobre su manera de aproximarse a informaciones de este tipo, que podría quizá ser más determinante a la hora de seleccionar a vuestra persona principa.